otter/test/utils.py

```diff
@@ -899,6 +899,18 @@ def noop(_):
     pass
 
 
+def const(v):
```

Comment:
Very nice!  I've been thinking about adding a function like this for a while!  Thank you, it will be very useful!! :)